shenyu-admin/src/main/java/org/apache/shenyu/admin/model/event/metadata/BatchMetaDataDeletedEvent.java

```diff
@@ -44,16 +42,7 @@ public BatchMetaDataDeletedEvent(final Collection<MetaDataDO> source, final Stri
         super(source, null, EventTypeEnum.META_DATA_DELETE, operator);
         this.deletedIds = ListUtil.map(source, BaseDO::getId);
     }
-    
-    @Override
-    public String buildContext() {
-        final String metaData = ((Collection<?>) getSource())
-                .stream()
-                .map(s -> ((MetaDataDO) s).getAppName())
-                .collect(Collectors.joining(","));
-        return String.format("the meta data [%s] is %s", metaData, StringUtils.lowerCase(getType().getType().toString()));
```

Comment:
why remove?

Reply:
Because its parent class method implements the same logic